pages/k8s/cni-multus.md

```diff
@@ -0,0 +1,225 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "CNI with Multus"
+  description: How to deploy and use Multus in Charmed Kubernetes
+keywords: CNI, networking, multus
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: cni-multus.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+[Multus][multus] is a CNI plugin for Kubernetes that enables attaching multiple
+network interfaces to pods. Multus support for **Charmed Kubernetes** is
+provided as a Kubernetes charm.
+
+## Requirements
+
+### Juju 2.8 features
```

Comment:
We'll want to keep this section up-to-date as the Juju 2.8 release makes its way through beta, and eventually to stable, over the next couple of months.